### ISSUE TITLE
Multiple testing correction strategy

### ISSUE THREAD

**Issue by ielis** [monarch-initiative/gpsea#126]
Peter's info:

---

I think we need a more general interface than just MC type, because we will also want to choose which HPO terms to test using flexible criteria. Probably something like

```
class MultipleCorrectionStrategy

    Attributes
    MC type (Bonferroni, ....)
    Min annot (int) - e.g., do not test if just one patient is annotated to a term
    hierarchy strategy (probably an enum) – what should we do with testing terms that are parent/child of each other? There are some strategies that we should definitely use. For instance, if the parent has the same annots as the child, just test the child.
```

There are more sophisticated ways of doing this, and I spoke with a statistician who might want to help us with this, but it would be good to implement the above and then send it to Mike Love for possible collaboration.

**Comment by ielis** [comment]
We need to prepare a notebook that presents the nuts and bolts of the analysis.

We need to extract the code that runs the G/P analysis into a separate class and present it in the notebook.

**Comment by pnrobinson** [comment]
See https://github.com/monarch-initiative/genophenocorr/pull/160

**Comment by pnrobinson** [comment]
I think this was closed by #160
